app/src/main/AndroidManifest.xml

```diff
@@ -1,21 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.onboarding.marvelapp">
+    package="com.marvelapp">
 
     <application
+        android:name=".presentation.application.MarvelApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.MarvelApp">
-        <activity android:name=".MainActivity">
+        android:theme="@style/Theme.Design.NoActionBar">
+        <activity
+            android:name=".presentation.activity.SplashActivity"
+            android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity
+            android:name=".presentation.activity.MainMenuActivity"
+            android:exported="false" />
+        <activity
+            android:name=".presentation.activity.CharacterActivity"
+            android:exported="false" />
     </application>
 
 </manifest>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
Solved